### DISCUSSION TITLE
Remote Docker Deployment

### DISCUSSION THREAD

**Discussion by nanikjava** [YuukanOO/seelf#78] [General]
Hi,

Can't find any doc on deploying on remote docker but tried but always get the following 
![image](https://github.com/user-attachments/assets/21f6f28a-7c54-476b-9437-86b99b8ea891)

I've setup the SSH private key in seelf admin

Any idea ?

**Comment by YuukanOO** [comment]
If the configuration has failed it means something went wrong when trying to connect to the remote host to deploy the needed infrastructure.

Your target should appears with a red border (in the targets page) and the error message should show at the top of your target edit page.

**Reply by YuukanOO** [reply]
Is your issue resolved @nanikjava ?

**Reply by nanikjava** [reply]
Yes, issue was fixed. I have to resetup again and it works. Thanks
